### PR TITLE
Move target OS to iOS 13.0

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: Parent, Teacher, Student
release note: Dropped support for iOS 12

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49741254/p/002b65c0605694932e741130245abc67">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49741273/p/bdf117a34fac0fe7cf24d0f616e1d7f0">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49741233/p/abd1d88b585ea2c38ef66356590d75d4">
    
  </a>
</details>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.17% | 93.17% | 0%

#### Affected Apps: Parent, Teacher, Student

#### Release Note: 
Dropped support for iOS 12
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/UIViews/CoreWebView.swift [hunk 1]
```diff
@@ -257,6 +257,9 @@ open class CoreWebView: WKWebView {
                 document.addEventListener('load', checkSize, true)
             })
             window.addEventListener('error', checkSize, true)
+            if (window.ResizeObserver) {
+                new ResizeObserver(checkSize).observe(document.documentElement)
+            }
```
Comment (vanwagonet) [inline Core/Core/UIViews/CoreWebView.swift:262]: This fixes a bug in the size I saw in a particular discussion. It's only iOS 13.4+, so I didn't check to see if it obsoletes any of the other mechanisms.

File: Student/Student.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2161,7 +2157,6 @@
 				495D8BB91CFE17D700D41098 /* LaunchScreen.storyboard in Resources */,
 				7D38A7D724CF610E00A4373E /* QuizDetailsViewController.storyboard in Resources */,
 				7DCF0FA522F8A8A9000AACCE /* QuizListViewController.storyboard in Resources */,
-				B5F122C91D8C4D6C0045063A /* Student.entitlements in Resources */,
```
Comment (vanwagonet) [inline Student/Student.xcodeproj/project.pbxproj:2164]: There was a warning about this not needed to get copied into the bundle, so I removed it.

File: Student/Student.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2577,7 +2572,6 @@
 				E80936E424C24CDD00D15E9A /* CourseGroupE2ETests.swift in Sources */,
 				E8C6F4E223D626220086B865 /* PeopleE2ETests.swift in Sources */,
 				E8DB184F24C7883B0042D6D7 /* DashboardE2ETests.swift in Sources */,
-				E80936E524C24CDD00D15E9A /* DashboardE2ETests.swift in Sources */,
```
Comment (vanwagonet) [inline Student/Student.xcodeproj/project.pbxproj:2580]: duplicate

**Review by armstrongnate — APPROVED** [review]
LGTM 🥳

**Pull request merged**